### PR TITLE
Change SendEmail action

### PR DESCRIPTION
**Pull request by vova07**

### CODE REVIEW

**Comment by vova07** [comment]
please review

File: spec/definitions/Rules/RuleAction.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ properties:
       - schedule-payment-retry
       - schedule-payment
       - send-email
+      - split-test-email
```
Comment (adamaltman) [inline spec/definitions/Rules/RuleAction.yaml]: As we discussed, this will change back to `send-email` at some point.

**Comment by vova07** [comment]
please re-review

File: spec/definitions/Rules/RulesEmailNotification.yaml [hunk 2]
```diff
@@ -58,6 +58,11 @@ properties:
           description: The attachment's resource identifier string. The template placeholders are allowed.
           type: string
           example: "INV-1"
+  weight:
+    type: integer
+    description: The message's weight.
+    example: 75
+    default: 100
```
Comment (slavcodev) [inline spec/definitions/Rules/RulesEmailNotification.yaml]: what about minimum value?

**Comment by vova07** [comment]
please re-review

**Pull request merged**